update_compilers/install_binaries.sh

```diff
@@ -49,15 +49,18 @@ if [[ ! -d ${OPT}/pahole ]]; then
     pushd /tmp/build
 
     # Install elfutils for libelf and libdwarf
-    fetch https://sourceware.org/elfutils/ftp/0.175/elfutils-0.175.tar.bz2 | tar jxf -
-    pushd elfutils-0.175
-    ./configure --prefix=/opt/compiler-explorer/pahole --program-prefix="eu-" --enable-deterministic-archives
+    fetch https://sourceware.org/elfutils/ftp/0.179/elfutils-0.179.tar.bz2 | tar jxf -
```

Comment:
This looks OK to me but we'll need to `rm -rf /opt/compiler-explorer/pahole` to get this to run: we don't check the version of `pahole` et al (and we probably should!)